variants/arduino_101/linker_scripts/flash.ld

```diff
@@ -52,7 +52,7 @@ __firq_stack_size = 1024;
 
 /* Minimum heap size to allocate
  * Actual heap size might be bigger due to page size alignment */
-__HEAP_SIZE_MIN = 8192;
+__HEAP_SIZE_MIN = 16384;
```

Comment:
I think we can add another 8K RAM for heap. Because ~8K RAM for user sketch may enough and now we have 40K RAM